internal/pkg/service/common/etcdop/iterator/iterator_test.go

```diff
@@ -489,12 +489,18 @@ func TestIterator_ForEach(t *testing.T) {
 	// Run op
 	assert.NoError(t, getAllOp.Do(ctx).Err())
 
+	// Clear loaded KVs before assert
+	operations := tracker.Operations()
+	for i := range operations {
+		operations[i].KVs = nil
+	}
+
```

Comment:
There were some update of the `etcdop.Tracer`, so it now traces also loaded KVs, so we need clear them, in some tests.